### PR TITLE
PPS NANOAOD dependency hotfix

### PR DESCRIPTION
**Pull request by grzanka**
#### PR description:

This PR resolves an issue reported by @mariadalfonso in https://github.com/cms-sw/cmssw/issues/39197#issuecomment-1228305691
Production of proton tables in PPS NANOAOD builders used a filtering method (set of Proton POG quality checks) which was implemented as an EDProducer located in `RecoCTPPS/ProtonReconstruction/plugins/PPSFilteredProtonProducer.cc`.
With that modification this filtering is disabled, removing dependencies on `RecoCTPPS/ProtonReconstruction`, instead proton data is directly taken from MINIAOD collection.

We foresee adding in the future another modification which enables to use of such filtering in an on-demand scenario and with proper dependencies. 

#### PR validation:

The limited battery of tests shows no errors (`runTheMatrix`).
Impact on the NANOAOD production was cross-checked by the proton POG group (@diemort )

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

This is not a backport.

### CODE REVIEW

File: PhysicsTools/NanoAOD/python/protons_cff.py [hunk 1]
```diff
@@ -2,26 +2,18 @@
...
 
-filteredProtons = ppsFilteredProtonProducer.clone(
-    protons_single_rp = cms.PSet(
-        include = cms.bool(singleRPProtons)
-    )
-)
-
 protonTable = cms.EDProducer("ProtonProducer",
-                             tagRecoProtonsMulti  = cms.InputTag("filteredProtons", "multiRP"),
+                             tagRecoProtonsSingle = cms.InputTag("filteredProctppsProtonstons", "singleRP"),
```
Comment (forthommel) [inline PhysicsTools/NanoAOD/python/protons_cff.py:9]:
```suggestion
                             tagRecoProtonsSingle = cms.InputTag("ctppsProtons", "singleRP"),
```

File: PhysicsTools/NanoAOD/python/protons_cff.py [hunk 2]
```diff
@@ -2,26 +2,18 @@
...
+                             tagRecoProtonsMulti  = cms.InputTag("filteredProctppsProtonstons", "multiRP"),
```
Comment (forthommel) [inline PhysicsTools/NanoAOD/python/protons_cff.py:10]:
```suggestion
                             tagRecoProtonsMulti  = cms.InputTag("ctppsProtons", "multiRP"),
```

**Review by forthommel — CHANGES_REQUESTED** [review]

**Comment by perrotta** [comment]
please test

**Comment by forthommel** [comment]
It does not seem like the tau module failure (`pat::Tau: the ID byVVVLooseDeepTau2018v2p5VSjet can't be found in this pat::Tau.`) is any related to this PR, right?

**Comment by grzanka** [comment]
@vlimant what is the status of the integration testing ?

**Comment by vlimant** [comment]
+1
all changes now look sensible

File: PhysicsTools/NanoAOD/python/protons_cff.py [hunk 3]
```diff
@@ -58,7 +50,7 @@
     ),
 )
 
-protonTablesTask = cms.Task(filteredProtons,protonTable,multiRPTable)
+protonTablesTask = cms.Task(protonTable,multiRPTable)
```
Comment (perrotta) [inline PhysicsTools/NanoAOD/python/protons_cff.py:53]: (Checking): shouldn't `ctppsProtons` be added to the task in place ot `filteredProtons` here?

Comment (perrotta) [reply PhysicsTools/NanoAOD/python/protons_cff.py:53]: > (Checking): shouldn't `ctppsProtons` be added to the task in place ot `filteredProtons` here?

@grzanka

Comment (forthommel) [reply PhysicsTools/NanoAOD/python/protons_cff.py:53]: @grzanka please confirm, but I believe this task should only include `protonTable` and `multiRPTable`. @perrotta, `ctppsProtons` is actually the sequence we would like to strip from the NANOAODification.

**Comment by perrotta** [comment]
+1

**Pull request merged**